lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -18,22 +18,29 @@ define((require, exports, module) ->
             span = $("<span>").attr("id", "formula")
             $("body").append(span)
 
-            katex = require("ace/ext/katex")
+            require(["ace/ext/katex/katex"], (katexInner) ->
+                katex = katexInner
+                return
+            )
+            return
+
+        callback = (editor) ->
+            unless katex?
+                initKaTeX()
+                setTimeout((-> callback(editor)), 0)
```

Comment:
is it a hack for waiting until katex loads? I suggest passing code which should be executed to `initKatex` instead